src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,32 @@
-export const PostInfo = () => <>Put the post here</>;
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => {
+  const { title, user, body, comments } = post;
+  const CommentLength =
+    comments.length === 0 ? (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    ) : (
+      <CommentList comments={comments} key={comments.id} />
```

Comment:
According to the checklist, a `key` should only be added to the root tag inside an iteration. Since `CommentList` is not being iterated over here, the `key` prop should be removed.